cmd/build/helmify/replacements.go

```diff
@@ -119,6 +119,24 @@ var replacements = map[string]string{
     {{- end }}
     resources:
     - '*'
+    # Explicitly list all known subresources except "status" (to avoid destabilizing the cluster and increasing load on gatekeeper).
+    # You can find a rough list of subresources by doing a case-sensitive search in the Kubernetes codebase for 'Subresource("'
+    - 'pods/ephemeralcontainers'
```

Comment:
Thank you for adding this @mac-chaffee 🎉

Do we want to create a parameter for this so people can opt-in or opt-out? Should this be turned on by default?

Reply:
I read @maxsmythe 's reasoning [here](https://github.com/open-policy-agent/gatekeeper/pull/1806#issuecomment-1020747482) to mean that the default should not risk under-enforcement; the current default is not a "conservative" one from a security perspective. There is a performance hit, but I think it's far more likely a user will think `*` covers everything than for a user to blindly roll out this next version to a big cluster which happens to call subresources significantly more than regular resources.

Plus, advanced users can already override this whole field anyway due to #1806 .